test/helpers/coreHelpers.ts

```diff
@@ -164,6 +164,18 @@ export const deployTokensAsync = async (
   tokenCount: number,
   provider: Provider,
   owner: Address = DEFAULT_ACCOUNT,
+): Promise<StandardTokenMockContract[]> => {
+  const decimals: number[] = [];
+  _.times(tokenCount, () => decimals.push(_.random(4, 18)));
+
+  return deployTokensSpecifyingDecimals(tokenCount, decimals, provider, owner);
+};
+
+export const deployTokensSpecifyingDecimals = async (
```

Comment:
Can you build this into `deployTokensAsync` instead of copying and pasting `deployTokensAsync` and adding a param? Also in the future, less awkward naming. `deployTokensWithDecimals`